docs/release_notes.md

```diff
@@ -6,20 +6,24 @@
 
 - Attachment to geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
 - Slide on geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
+- Shape preservation constraints integrated in all solvers.
 - Point mass support integrated in all solvers to control the dynamics of the system.
+- Compute point mass distribution based on the density and volume or area.
```

Comment:
I would not add the specifier about volume or area. No need to expose the internals
```suggestion
- Compute point mass distribution based on the density.
```